membership-common/src/main/scala/com/gu/memsub/subsv2/Plan.scala

```diff
@@ -237,10 +238,12 @@ object CatalogPlan {
   type Partner[+B <: BillingPeriod] = CatalogPlan[Product.Membership, PaidCharge[Partner.type, B], Current]
   type Patron[+B <: BillingPeriod] = CatalogPlan[Product.Membership, PaidCharge[Patron.type, B], Current]
 
-  type Contributor = CatalogPlan[Product.Contribution, PaidCharge[Contributor.type, Month.type], Current]
+  type Contributor[+B <: BillingPeriod] = CatalogPlan[Product.Contribution, PaidCharge[Contributor.type, B], Current]
 
   type Digipack[+B <: BillingPeriod] = CatalogPlan[Product.ZDigipack, PaidCharge[Digipack.type, B], Current]
-  type SupporterPlus[+B <: BillingPeriod] = CatalogPlan[Product.SupporterPlus, PaidCharge[SupporterPlus.type, B], Current]
+
+  type SupporterPlus[+B <: BillingPeriod] = CatalogPlan[Product.SupporterPlus, PaidChargeList, Current]
```

Comment:
Did you change this to use `PaidChargeList ` or was this left over from my change?  The other instance of this uses `SupporterPlusContribution`

Reply:
ah right, hence why it compiled fine, I thought I was using the class you made in here.